### DISCUSSION TITLE
Using multiple predictors

### DISCUSSION THREAD

**Discussion by skilpinen** [Thie1e/cutpointr#46] [Q&A]
Hi,

This is brilliant library, however can you advice what is recommended use of the library in case of multiple predictors?

I have relatively simple case of having two predictors (one good, one not so good but still valuable) which I use to predict simple binary decision. I have looked at multi_cutpointr and understand results of that. However, what is the suggested use of multiple predictors? I mean that predict() does not have method for multi_cutpointr object. In other words, in the context of cutpointr are there any functionalities to tie these predictors together and use both with predict()? Or what is the suggested plan in this case? Or is this then like multivariable logistic regression case and out of context of cutpointr.

**Comment by Thie1e** [comment]
Hi, well, that's the question really: What is the desired end result of your analysis? 

1. If you need separate cutoffs and classification statistics for the two predictors, you can use multi_cutpointr. But multi_cutpointr can not incorporate multiple predictors into a single model and create predictions from that, because that's not the goal of the package. As you have correctly assumed.
2. Do you want a model that incorporates multiple variables for prediction? Then, as you said, you need something like logistic regression. If you then need statistics for binary classification or statistics regarding cutoffs on the predicted probabilities from the logistic regression, you can use cutpointr.
